pom.xml

```diff
@@ -18,10 +18,9 @@
 		<module>spring-cloud-starter-dataflow-server-kubernetes</module>
 	</modules>
 
-
 	<properties>
 		<spring-cloud-dataflow.version>1.0.0.RELEASE</spring-cloud-dataflow.version>

```

Comment:
Wasn't sure if I should bump `spring-cloud-dataflow.version` to `1.0.1.BUILD-SNAPSHOT` as well?